packages/create-bison-app/template/src/components/auth/LoginForm.tsx

```diff
@@ -91,11 +90,11 @@ export function LoginForm() {
         </PasswordField>
       </div>
       <div className="flex justify-between">
-        <NextLink href="/reset-password">
-          <Button variant="link" size="sm">
+        <Link href="/reset-password">
+          <Button variant="link" size="sm" type="button">
```

Comment:
If the user hits enter on their keyboard, it redirected them to `/reset-password` instead of signing in. The workaround is having `type=button` attribute. This is necessary when there are multiple buttons in a form.